setup.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+import pathlib
+
+from setuptools import setup
+from setuptools.command.build_py import build_py
+
+
+# Adapted from https://stackoverflow.com/a/71137790
+class build_py_with_redirector(build_py):  # noqa: N801
+    """Include the redirector files in the generated wheel."""
+
+    def copy_redirector_file(self, extension):
+        redirector = f"_numba_cuda_redirector.{extension}"
+        source = pathlib.Path("site-packages") / redirector
```

Comment:
Nitpick: specializing the function on the filename or the path this way isn't my favorite. I would rather see this function accept two arguments, 1) the input file path relative to the package root and 2) the target directory. The latter can default to "." so that the destination defaults to `self.build_lib` and you can call the function with just the input filename below.

Reply:
Changed to follow the suggestion, and verified locally that that change didn't result in a change to the built wheel.